scripts/directtools/__init__.py

```diff
@@ -751,14 +754,25 @@ def plotSofQW(workspace, QMin=0., QMax=None, EMin=None, EMax=None, VMin=0., VMax
         colorbar.set_label(r"$\chi''(Q,E)$ (arb. units)")
     else:
         colorbar.set_label(r'$S(Q,E)$ (arb. units)')
-    axes.set_xlim(left=QMin)
-    axes.set_xlim(right=QMax)
-    axes.set_ylim(bottom=EMin)
-    if EMax is not None:
-        axes.set_ylim(top=EMax)
-    axes.set_xlabel(u'$Q$ (\u00c5$^{-1}$)')
-    axes.set_ylabel('Energy (meV)')
-    _SofQWtitle(workspace, figure)
+    if qHorizontal:
+        xLimits = {'left': QMin, 'right': QMax}
+        yLimits = {'bottom': EMin}
+        if EMax is not None:
+            yLimits['top'] = EMax
+        xLabel = u'$Q$ (\u00c5$^{-1}$)'
```

Comment:
```suggestion
        xLabel = u'$Q$ ($\AA^{-1}$)'
```

Reply:
I did not use the `\AA` notation originally because math symbols are in *italics* while the units are not. However, now I found a way to force the correct font face. This should simplify some strings considerably so thanks for the hint!